packages/form-js/CHANGELOG.md

```diff
@@ -6,6 +6,29 @@ All notable changes to [form-js](https://github.com/bpmn-io/form-js) are documen
 
 ___Note:__ Yet to be released changes appear here._
 
+## 0.2.0
+
+### General
+
+* `CHORE`: refactor for extensibility and testability
+* `CHORE`: separate component instantation and import
+
+### Viewer
+
+* `FEAT`: improve robustness of form import ([#79](https://github.com/bpmn-io/form-js/issues/79))
+* `FEAT`: allow programmatic form validation ([#30](https://github.com/bpmn-io/form-js/issues/30))
+* `FEAT`: prevent submission of readonly form
+* `FIX`: correct submission of empty fields ([#54](https://github.com/bpmn-io/form-js/issues/54))
+* `FIX`: correct submission of disabled fields ([#6](https://github.com/bpmn-io/form-js/issues/6))
+
+### Editor
+
+* `FEAT`: add undo and redo ([#45](https://github.com/bpmn-io/form-js/issues/45))
+
+### Breaking Changes
+
+* Form viewer and editor import are now asynchronous. Creating either via `#createForm` and `#createFormEditor` must be awaited.
```

Comment:
Edtior import is still synchronous. It's what https://github.com/bpmn-io/form-js/issues/82 is about.

Reply:
Idea: Make it async, fake it async or pretend it's async.

Reply:
Pull Request: https://github.com/bpmn-io/form-js/pull/91